File: notifier/src/main/java/com/juanimoli/strictmodenotifier/ReportActivityUtils.java
package com.juanimoli.strictmodenotifier;

import android.app.ActionBar;
import android.app.Activity;

/**
 * @author nishimura_shinichi
 */
class ReportActivityUtils {
    public static void setTitle(Activity activity, String title) {
        ActionBar actionBar = activity.getActionBar();
        //noinspection ConstantConditions
        actionBar.setTitle(title);
    }

    public static void setDisplayHomeAsUpEnabled(Activity activity, boolean enabled) {
        ActionBar actionBar = activity.getActionBar();
        //noinspection ConstantConditions
        actionBar.setDisplayHomeAsUpEnabled(enabled);
    }
}

File: notifier/src/main/java/io/github/juanimoli/strictmode/notifier/ReportActivityUtils.kt
package io.github.juanimoli.strictmode.notifier

import android.app.Activity

/**
 * @author nishimura_shinichi
 */
internal object ReportActivityUtils {
    fun setTitle(activity: Activity, title: String?) {
        val actionBar = activity.actionBar
        actionBar!!.title = title
    }

    fun setDisplayHomeAsUpEnabled(activity: Activity, enabled: Boolean) {
        val actionBar = activity.actionBar
        actionBar!!.setDisplayHomeAsUpEnabled(enabled)
    }
}
